pilot/pkg/bootstrap/namespacecontroller.go

```diff
@@ -141,26 +135,3 @@ func (nc *NamespaceController) insertDataForNamespace(ns string) error {
 	}
 	return certutil.InsertDataToConfigMap(nc.client, meta, nc.getData())
 }
-
-// On namespace change, update the config map.
-// If terminating, this will be skipped
-func (nc *NamespaceController) namespaceChange(obj interface{}) error {
-	ns, ok := obj.(*v1.Namespace)
-
-	if ok && ns.Status.Phase != v1.NamespaceTerminating {
```

Comment:
we are no longer handling ns termination. Will this lead to errors when a namespace is deleted?

Reply:
When a namespace is deleted the logs of Istiod will be spammed with a bunch of errors because we attempt to write to the deleted namespace

Reply:
With this PR, the namespace controller only handles the events of adding a namespace and updating a namespace. Thus, when a namespace is deleted, there will be no error logs of attempting to write the configmap to the deleted namespace.

Reply:
Lei, I have deployed this pr to a cluster and tried it. A delete triggers the update callback as well

Reply:
Thanks! A check of namespace terminating is added as follows:
if ok && ns.Status.Phase != v1.NamespaceTerminating {
...
}